### PR TITLE
Update File Rename sheet window.

### PR DESCRIPTION
**Pull request by nevack**
Fix deprecations warnings.
Use modern ObjectiveC features.

### CODE REVIEW

**Comment by nevack** [comment]
Hi, @livings124, may I ask you to review my PR?

File: macosx/FileRenameSheetController.h [hunk 1]
```diff
@@ -12,17 +12,6 @@
 @class Torrent;
 
 @interface FileRenameSheetController : NSWindowController
-{
```
Comment (livings124) [inline macosx/FileRenameSheetController.h:15]: Can I recommend doing a separate PR that removes all of these 32-bit variables (in a bunch of different files)?

File: macosx/FileRenameSheetController.xib [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.Cocoa.XIB" version="3.0" toolsVersion="11762" systemVersion="16D32" targetRuntime="MacOSX.Cocoa" propertyAccessControl="none">
```
Comment (livings124) [inline macosx/FileRenameSheetController.xib:2]: What's changing in this xib?

File: macosx/FileRenameSheetController.h (hunk 1 above)
Comment (nevack) [reply macosx/FileRenameSheetController.h:15]: Ok, I'll try.

File: macosx/FileRenameSheetController.xib (hunk 2 above)
Comment (nevack) [reply macosx/FileRenameSheetController.xib:2]: Nothing really, just touched by Xcode, I think it converted XIB to autolayout.
Let's decline this PR in favor in ivar convertion.

**Pull request closed without merging**